src/lib/modules/blockchain_connector/provider.js

```diff
@@ -16,9 +20,34 @@ class Provider {
 
   startWeb3Provider(callback) {
     const self = this;
-
+    
     if (this.type === 'rpc') {
       self.provider = new this.web3.providers.HttpProvider(self.web3Endpoint);
+    } else if (this.type === 'ledger') {
+        self.accountsConfig.forEach((accountConfig) => {
```

Comment:
We should print an error if there is no `hardwareWallet: 'ledger'` inside the accountsConfig, since we have no way to connect to it.

Also, what should we do if a user puts two accounts with a wallet config? We should at least warn and pick only the first one.

Reply:
@jrainville done, thanks again for the great feedback 🙂